dev/lint-scala

```diff
@@ -29,14 +29,14 @@ ERRORS=$(./build/mvn \
     -Dscalafmt.skip=false \
     -Dscalafmt.validateOnly=true \
     -Dscalafmt.changedOnly=false \
-    -pl connector/connect \
+    -pl connector/connect/server \
```

Comment:
For this case, we need official JIRA ID instead of `MINOR`. Could you file one, @dengziming ?